docs/cookbook/building-navigation.md

```diff
@@ -261,14 +261,16 @@ The final step is to turn the array data from this into template markup. We will
         {% for item in data.global.primaryNav %}
           <li>
             {% set path = '' %}
+            {% set pageTitle = '' %}
             {% if item.type === 'page' and item._page and item._page[0] %}
-              {% path = item._page[0]._url %}
+              {% set path = item._page[0]._url %}
+              {% set pageTitle = item._page[0].title %}
             {% elif item.type === 'custom' %}
-              {% path = item.url %}
+              {% set path = item.url %}
             {% endif %}
             <a href="{{ path }}"
               {% if item.target[0] === '_blank' %} target="_blank" {% endif %}
-            >{{ item.label }}</a>
+            >{{ item.label or pageTitle }}</a>
```

Comment:
This wasn't a typo, but we aren't requiring label and don't have a fallback to the page title when it's not used.